docker/exec.go

```diff
@@ -0,0 +1,67 @@
+package docker
+
+import (
+	"fmt"
+	"go.uber.org/zap"
+	"io"
+	"os"
+	"os/exec"
+)
+
+
+var (
+	logger *zap.SugaredLogger
+)
+
+func init() {
+	var err error
+	devLogger, err := zap.NewDevelopment()
```

Comment:
i dont think go-utils should initialize logging. especially not a development logger.
the main problem with dev log, is that DPanic will panic if we use it in prod.
but in generated i think we should avoid non-main packages initializing logging

Ideally the package should receive the logger from the context, or as a parameter.